### DISCUSSION TITLE
Getting Error [HTTP]:500 - [CorrelationId]:36e23ea0-502f-4000-1c31-a2ec77a8ad7c [Version]:16.0.0.22504 when searching large document Library

### DISCUSSION THREAD

**Discussion by GerritGoudsmit** [microsoft-search/pnp-modern-search#2100] [General]
I use the PnP Modern Search in the version 4.5.4.

Everything (i.e. search results and filters) is working fine, as long as I use the Modern Search for small to medium sized document libraries. As soon as I use the search for a large document library (in this case more than 37'000 elements), I get the following error.

[HTTP]:500 - [CorrelationId]:36e23ea0-502f-4000-1c31-a2ec77a8ad7c [Version]:16.0.0.22504

I guess that this issue arizes because of the large array of results. Is there any possibility to limit the number of the values in the Search Results web part? I cannot limit the number of the results by the Search Filters, as the should be able to define the filters by himself.

![Modern Search](https://user-images.githubusercontent.com/105798722/169280708-d9452e3c-84ce-44b0-b5ce-80d3607e4917.png)

**Comment by wobba** [comment]
Seems like an API issue. The number of results in a lib should not matter as search will max return 500 items per page. Also transferring to discussions as the issue template is not followed to provide setup information

**Comment by clmmlc** [comment]
I am having the same issue. Did you find a solution for this? I have found this only happens when I use a crawled property term in the search bar. If I use a general search term the results do not give me this error

![craweled property term](https://user-images.githubusercontent.com/107613292/174060914-1ac814fa-6f3d-4743-bbd1-c234791746f1.png)
![general term](https://user-images.githubusercontent.com/107613292/174060919-9ae65369-5b66-4619-b70d-7513e8c4885f.png)

**Comment by GerritGoudsmit** [comment]
In the meantime I found out what caused the error messages in my case. As wobba wrote it is not the size of the libraries. One of the libraries has now more than 200'000 elements and the Modern Search is working fine.

The error was caused by the filter "Author" in combination with "ModifiedBy". As soon as I removed the filter "Author" the Modern Search did not show any error messages anymore. In order to verify this, I recoomend to first remove all filters and then add one after the other and check for errors.

**Reply by wobba** [reply]
There has also been an API issue with refiner filters which has been fixed and is rolling out.

**Reply by clmmlc** [reply]
Oh great - thank you very much!
